client/lib/feed-stream-store/index.js

```diff
@@ -54,6 +54,27 @@ function buildNamedKeyMaker( property ) {
 	};
 }
 
+const conversationsKeyMaker = ( function() {
+	const baseMaker = buildNamedKeyMaker( 'last_comment_date_gmt' );
+	return function keyMaker( post ) {
+		const key = baseMaker( post );
+		key.comments = post.comments;
+		return key;
+	};
+} )();
+
+function conversationsPager( params ) {
+	delete params.meta;
+	delete params.orderBy;
+	if ( params.before ) {
```

Comment:
you can drop the if statements here always run the delete.
deleting a prop that doesn't exist runs just fine